src/plugins/ui/selection.ts

```diff
@@ -178,6 +178,7 @@ export class GridSelectionPlugin extends UIPlugin {
         break;
     }
     this.setSelectionMixin(event.anchor, zones);
+    this.selection.resetDefaultAnchor(this, deepCopy(this.gridSelection.anchor));
```

Comment:
note to self.   Since ResetDefaultAnchor is called at the very end of the stack, we might as well have SelectionProcessor do it itself when it dispatched its commands.

Reply:
After consideration - it seems quite hard to accomplish in stable because we need to update he value after both handling a selection modification AND a dispatched command.  Registering the selection processor to listen to dispatch looks really weird and looks like it would create a loop since it already uses model.render in its lifecycle